docs/src/installation.rst

```diff
@@ -206,6 +212,52 @@ using this compiler
 
 
 
+Build slsDetectorGui (Qt5)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Using pre-built binary on conda
+    .. code-block:: bash
+
+        conda create -n myenv slsdetgui=7.0.0
+        conda activate myenv
+
+
+2. Using system installation on RHEL7
+    .. code-block:: bash
+
+        yum install qt5-qtbase-devel.x86_64
+        yum install qt5-qtsvg-devel.x86_64 
+
+
+3. Using conda
+    .. code-block:: bash
+
+        #Add channels for dependencies and our library
+        conda config --add channels conda-forge
+        conda config --add channels slsdetectorgroup
+        conda config --set channel_priority strict
+
+        # create environment to compile
+        # on rhel7
+        conda create -n slsgui zeromq gxx_linux-64 gxx_linux-64 mesa-libgl-devel-cos6-x86_64 qt
+        # on fedora or newer systems
+        conda create -n slsgui zeromq qt
```

Comment:
in case you use conda compilers you would also need the libgl, but no need for it on fedora unless maybe using it with ROOT

Reply:
done